CONTRIBUTING.md

```diff
@@ -17,12 +17,36 @@ want a `.war` as quick as possible, you can use this instead:
 $ mvn -Dmaven.test.skip=true clean package 
 ```
 
+
 # Suggesting modifications
 
 Airsonic's source code is hosted on [github](https://github.com/airsonic/airsonic/),
 who provides a [lot of documentation](https://help.github.com/en) on how
 to contribute to projects hosted there.
 
+Keep in mind that we are a small project, and that we value stability over new
+features.
+
+
+# Getting your pull-requests reviewed an merged
+
+Once you issued a pull-request, it will be reviewed by other contributors, and
```

Comment:
This might be a promise that we can't keep. You touch on it below a bit, but if a PR's quality (size, tests, code quality, etc) is below an arbitrary line, it might not be reviewed by anyone..

Reply:
Additional text suggested:

Once you have submitted a pull-request, a number of factors may determine how much attention it receives, how quickly it may be reviewed, and whether or not it eventually gets accepted and merged.  Here are a few guidelines that can help speed the process and increase the chances of a pull request being accepted and merged in a timely fashion:

* Limit the scope to the minimum changes necessary to accomplish a discrete and well defined task.
* If your changes could be broken down into smaller units, then they are much less likely to be accepted.
* Try not to address unrelated issues in the same set of changes, unless failing to do so would cause other problems (like merge conflicts).
* Do your best to maintain prior functionality while eliminating (or at least minimizing) side effects.
* Maintain the style and coding standards represented by the codebase.
* Consistent, simple, and easy-to-understand changes are usually preferred.
* Do not mix functional changes with code cleanups or style changes.
* Make it as easy as possible on the maintainers to review your changes.  Rebasing your PR to address issues is preferred over adding additional commits that make changes to (or undo parts of) prior commits.
* In general, the more commits in a PR, the harder it is to review and the longer it will take to be reviewed.  But do not sacrifice good change isolation by combining commits unnecessarily.
* If your PR needs more than 2 or 3 commits, then you probably need to reduce the scope of your PR.  If a single commit touches more than a few files or more than 30-50 lines, then the scope of the commit probably needs to be reduced.

And finally:

* Actively maintain your PR.  If any concerns are raised, or tests fail, or some other change occurs in the codebase that affects your PR (like a merge conflict) before your PR is accepted and merged, you are expected to address and resolve those issues or the PR may be rejected.